synapse/lib/base.py

```diff
@@ -391,15 +391,12 @@ async def fini(self):
         for base in list(self.tofini):
             await base.fini()
 
-        try:
-            await self._kill_active_tasks()
-        except Exception:
```

Comment:
I want to say when this was originally written there was a situation where that _kill method could've thrown an exception but it could be obe at this point.

Reply:
This is probably handled by you catching Exception and CancelledErrror in the routine

Reply:
exactly